df2gspread/gfiles.py

```diff
@@ -77,7 +77,8 @@ def get_worksheet(gc, gfile_id, wks_name, write_access=False):
             wks = spsh.worksheet(wks_name)
         else:
             if write_access == True:
-                wks = spsh.add_worksheet(wks_name, 1000, 100)
+                rows, cols = new_sheet_dimensions
+                wks = spsh.add_worksheet(wks_name, rows, cols)
```

Comment:
Oh durf, right, args. Sure, I'll add basic checking but I assume it's perfectly acceptable to let gspread or Google handle the actual reporting of the error, if there is one, just like in the rest of the package.